### PR TITLE
Add support for Sentry native crashes and allow users to disable it or use their own DSN with client settings.

### PR DESCRIPTION
**Pull request by Pombal**
Upload debug symbols for nightly builds.
Add documentation for Ambient settings.

### CODE REVIEW

**Review by FredrikNoren — APPROVED** [review]
Nice!
For the settings doc to appear you need to update the SUMMARY.md file as well. 
Also there's a .github/release workflow which will also need the debug symbols (which we use to release point versions of ambient)

**Comment by Pombal** [comment]
@FredrikNoren Good catch!
I've updated the docs and moved the symbol upload to .github/release. I thought we were going to do releases from nightly builds and missed release.yml.

File: crates/settings/src/general.rs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    }
+
+    pub fn sentry_dsn(&self) -> String {
+        self.sentry.0 .1.clone()
+    }
+}
+
+// enabled, dsn
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub struct Sentry(pub (bool, String));
```
Comment (philpax) [inline crates/settings/src/general.rs:21]: I'd use explicit fields for this to make it obvious what they represent, especially for people editing the config

File: docs/src/user/settings.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Settings
+
+Ambient supports a number of settings that can be configured using the `settings.toml` file. This file is located under the platform's config directory, e.g. `C:\Users\*USER*\AppData\Roaming\Ambient\Ambient\config\settings.toml` on Windows.
+
+## Settings
+
+[general]\
```
Comment (philpax) [inline docs/src/user/settings.md:7]: You can use three-backticks followed by toml here, like here https://github.com/AmbientRun/Ambient/blob/main/docs/src/reference/models.md?plain=1#L8-L12

**Review by philpax — COMMENTED** [review]
LGTM, but a few minor tweaks before merge

**Pull request merged**